keras/layers/recurrent.py

```diff
@@ -1197,6 +1200,7 @@ def __init__(self, units,
                  dropout=0.,
                  recurrent_dropout=0.,
                  implementation=1,
+                 variant='reset_before',
```

Comment:
Seems like `reset_after=False` would be more user friendly (and consistent with the implementation). But "reset_after" could also be changed to something more explicit. Thoughts?

Reply:
Yeah, that sounds good. I'm not aware of any other conventions than those two, so a boolean might be sufficient.